examples/crowdfunding/test/crowdfundingTest.js

```diff
@@ -9,13 +9,14 @@ import {
 import { Runtime, StoreAccount } from '@algorand-builder/runtime';
 import { assert } from 'chai';
 
+const minBalance = 10000000; // 10 ALGO's
 const initialDonorBalance = 60000000;
-const initialCreatorBalance = 10000;
+const initialCreatorBalance = minBalance + 10000;
 const goal = 7000000;
 
 describe('Crowdfunding Tests', function () {
   let creator = new StoreAccount(initialCreatorBalance);
-  let escrow = new StoreAccount(0);
```

Comment:
when escrow is created, it has 0 balance right ? or is it deducted from creator ?

Reply:
yes, i think starting with 0 balance makes more sense, to maintain the minumum balance property i will donate 1 extra ALGO to escrow.

Reply:
no i wanted to ask what happens generally in TEAL code, in actual algorand network, when we create a escrow account, it start with zero right ? if yes then why there is minimum balance concept(or it excludes escrow ?)
if not what is the amount for escrow ?

Reply:
In actual algorand network, whenever we create an account, i has 0 balance (confirmed using goal). This is usually what we do in examples. (Whenever we create or load an account, before the actual transaction we 'fund' the account with some ALGO's by master account so that we don't get `overspend` error).

But with store accounts, we can initialize the account with some balance, so we're doing it directly.
The other way would be to initialize all store accounts with 0 balance, along with a master account (with a lot of balance), and fund those accounts by master using `algotransfer` txn (`runtime.transferAlgo(..)`).

Reply:
okay, just confirming because lsig is creating address account. new account created will have zer0 balance

Reply:
Ok